cpp/include/cudf/io/orc.hpp

```diff
@@ -547,11 +564,11 @@ class orc_writer_options {
   void set_compression(compression_type comp) { _compression = comp; }
 
   /**
-   * @brief Enable/Disable writing column statistics.
+   * @brief Choose granularity of statistics collection.
```

Comment:
Since this is the entry point for the users who will be changing the default behavior, might be good to somehow refer to the ORC frequency aliases defined above.

Reply:
Fixed documentation. Additionally, changed all sites where `stats_freq` is overwritten.